### PR TITLE
Do not use string interpolation in `where`

### PR DESCRIPTION
**Pull request by osyo-manga**
Do not use string interpolation in `where` !!!!
Used Arel instead.

### CODE REVIEW

**Comment by osyo-manga** [comment]
@meganemura @tkimi Please reviewing.

File: lib/activerecord-bitemporal/bitemporal.rb [hunk 1]
```diff
@@ -233,16 +233,16 @@ module Scope
           column_valid_to = "#{table_name}.valid_to"
           column_valid_from = "#{table_name}.valid_from"
           ignore_valid_datetime
-            .tap { |relation| break relation.where("? <= #{column_valid_to}", from.in_time_zone.to_datetime) if from }
-            .tap { |relation| break relation.where("#{column_valid_from} <= ?", to.in_time_zone.to_datetime) if to }
+            .tap { |relation| break relation.where(klass.arel_table[:valid_to].gteq(from.in_time_zone.to_datetime)) if from }
```
Comment (tkimi) [inline lib/activerecord-bitemporal/bitemporal.rb]: If so, I think `column_valid_to`  and `column_valid_from` are unnecessary.

Comment (osyo-manga) [reply lib/activerecord-bitemporal/bitemporal.rb]: Thanks! 
I removed it. https://github.com/kufu/activerecord-bitemporal/pull/35/commits/ea4d585fe11ea8f2f1ef772918f2c5e3ebbf47ae

**Comment by osyo-manga** [comment]
@meganemura @tkimi Thanks!!!!

**Pull request merged**